bogota-abril-2019/css-magazine-cover/Carlos David Diaz - Tarea Sesion 2/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="index.css">
+    <link href="https://fonts.googleapis.com/css?family=Basic|Holtwood+One+SC|Megrim" rel="stylesheet">
+    <title>Magazine Cover</title>
+</head>
+<body>
+    <div class="main-container">
+      <header class="banner">
+            <section id="slogan" class="subtitle highlight">
+                    <h3>Special Collector's Issue</h3>
+            </section>
+            <figure id="banner-logo">
+                <img src="assets/magazine-logo.png" alt="Magazine Logo" title="Magazine Cover">
+            </figure>
+            <section id="banner-info">
+                <h2>5 SONGS</h2>
+                <ul class="info-list">
+                    <li><span class="highlight">LED ZEPELLIN</span><br>"NO QUARTER"</li>
+                    <li><span class="highlight">STEVIE RAY VAUGHAN</span><br>"LENNY"</li>
+                    <li><span class="highlight">KID ROCK</span><br>"SO HOTT"</li>
+                    <li><span class="highlight">DIO</span><br>"HOLY DIVER"</li>
+                    <li><span class="highlight">HOLIDAY SPECIAL</span><br>"SILENT NIGHT"</li>               
+                </ul>
+            </section>
+        </header>
+        <aside class="news-container">
+            <article class="top-news">
+                <h1 class="big-title highlight">ROCK CONCERT</h1>
```

Comment:
Unnecessary used of Capital letter. Please use Text-transform CSS property.

https://css-tricks.com/almanac/properties/t/text-transform/